src/gauss_fit.jl

```diff
@@ -3,19 +3,21 @@ export gauss_line_fit
 
 function gaussclossure(X::AbstractVector)
     function gaussloss(params)
-        a, σ = params[1], params[2]
+        b, a, σ = params[1], params[2], params[3]
         x = a * [exp(-(t)^2 / (2 * σ^2)) for t in axes(X, 1)]
+        x .+= b
         return sum(abs2, x - X)
     end
     return gaussloss
 end
 
 function gauss_line_fit(X::AbstractVector; kwargs...)
     f = gaussclossure(X)
-    sX = sum(X)
-    σ² = sum(i^2 * X[i] for i in axes(X, 1)) / sX
+    mX = minimum(X)
```

Comment:
While it's slower, I might consider `median` rather than `minimum` here. One issue is that the PCO camera can use 4 pixels to encode a time stamp, and the content of those pixels is unrelated to image intensity; as a consequence, the value of `minimum` could at least theoretically be basically arbitrary. (Alternatively you can mask such pixels out, e.g., `minimum(view(X, inds))` where `inds` is chosen to exclude such pixels.) See https://www.pco.de/fileadmin/user_upload/pco-manuals/MA_PCOSDK_V127.pdf, and then search for PCO_GetTimestampMode. This feature is turned on by default in [Imagine](https://github.com/HolyLab/Imagine/blob/2d2b06f79bbdccfde84359c5233e98ae27462cf8/Imagine/cooke.cpp#L208-L209).